orangecontrib/text/nyt.py

```diff
@@ -51,7 +51,7 @@ def keywords(doc, name):
         (data.StringVariable('Creative Works'), lambda doc: NYT.keywords(doc, 'creative_works')),
         (tv, lambda doc: NYT.tv.parse(doc.get('pub_date'))),
         (data.DiscreteVariable('Article Type'), lambda doc: doc.get('type_of_material', None)),
-        (data.DiscreteVariable('Word Count'), lambda doc: doc.get('word_count', None)),
+        (data.StringVariable('Word Count'), lambda doc: doc.get('word_count', None)),
```

Comment:
But it's continuous, right? So it should be `ContinuousVariable`.

Add argument `number_of_decimals=0` to the `ContinuousVariable`'s constructor and it will show 0 decimals. If it doesn't, fix they way the value is printed out.